### PR TITLE
Add CHANGELOG entry for whitespace fixes

### PR DESCRIPTION
**Pull request by nickcolley**
Missed a CHANGELOG entry for https://github.com/alphagov/govuk-frontend/pull/1353

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -210,6 +210,13 @@
 
 🔧 Fixes:
 
+- Fix improperly indented html output in Design System examples
+
+  Thanks to [Debs](debsdee) and [Joe](joelanman) for raising this issue.
```
Comment (36degrees) [inline CHANGELOG.md]: These links 404 – suspect that's not what you're intending?

Comment (nickcolley) [reply CHANGELOG.md]: oopers

Comment (nickcolley) [reply CHANGELOG.md]: Fixed thanks.

File: CHANGELOG.md [hunk 2]
```diff
@@ -210,6 +210,13 @@
 
 🔧 Fixes:
 
+- Fix improperly indented html output in Design System examples
+
+  Thanks to [Debs](https://github.com/debsdee) and [Joe](https://github.com/joelanman) for raising this issue.
+
+([PR #1353](https://github.com/alphagov/govuk-frontend/pull/1353))
```
Comment (36degrees) [inline CHANGELOG.md]: Minor, but this should be indented so it sits with the rest of the bullet point. (Sorry, didn't notice this first time)

Comment (nickcolley) [reply CHANGELOG.md]: No worries, updated.

**Pull request merged**